### PR TITLE
docs(components-native): Add Mobile Typography docs

### PR DESCRIPTION
**Pull request by rodrigoeidelvein**
## Motivations

Adding the mobile `Typography` docs and moving them to the new folder and pattern.

### Added

- Add `Typography` mobile docs

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

**Review by MichaelParadis — CHANGES_REQUESTED** [review]
Overall looks good. I noticed that the alignment prop isn't actually changing the alignment
image


This does work on the Jobber Mobile storybook though

**Comment by rodrigoeidelvein** [comment]
@MichaelParadis great catch! I added the `SafeAreaProvider` to `SBProvider`.

File: docs/components/Typography/Mobile.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+<ArgsTable of={Typography} story="Mobile" />
+
+## Alignment
+
+<Canvas>
+  <Story
+    name="Alignment"
+    args={{
+      align: "right",
```
Comment (MichaelParadis) [inline docs/components/Typography/Mobile.stories.mdx:44]:
```suggestion
      align: "end",
```

**Review by MichaelParadis — CHANGES_REQUESTED** [review]
Minor change and then good to go

File: docs/components/Typography/Mobile.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+# Typography (Internal)
+
+<Banner type="notice" dismissible={false}>
+
+Typography is the component that powers our Heading and Text components “under
+the hood”. You should first look to use
+[Heading](/docs/atlantis-text-and-typography-heading--heading) and
+[Text](/docs/atlantis-text-and-typography-text--text) before you reach for
+Typography.
```
Comment (darryltec) [inline docs/components/Typography/Mobile.stories.mdx:15]: the same could be said for web so this should be under the docs story

File: docs/components/Typography/Mobile.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+</Banner>
+
+```ts
+import { Typography } from "atlantis/Typography";
```
Comment (darryltec) [inline docs/components/Typography/Mobile.stories.mdx:20]:
```suggestion
import { Typography } from "@jobber/components-native";
```

**Review by darryltec — CHANGES_REQUESTED** [review]
I would go and look at the similar things between the web and mobile component and document them under docs. If we don't know what those are then I'd suggest to skip this and come back to it in the future

File: docs/components/Typography/Typography.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+# Typography
+
+<Meta title="Components/Typography/Docs" component={Typography} />
+
+Base component that defines typographic standards across Jobber.
+
+Typography is the component that powers our Heading and Text components “under
+the hood”. You should first look to use
+[Heading](/docs/atlantis-text-and-typography-heading--heading) and
```
Comment (MichaelParadis) [inline docs/components/Typography/Typography.stories.mdx:13]: Final comment, I think update the links to go to `Heading` and the docs page for `Text`

Comment (rodrigoeidelvein) [reply docs/components/Typography/Typography.stories.mdx]: Good catch! I fixed it

**Review by MichaelParadis — APPROVED** [review]
LGTM

**Comment by darryltec** [comment]
i think we should just skip and close this PR. need to think about how to properly document this like Text or Icon

**Pull request closed without merging**